.gitignore

```diff
@@ -6,11 +6,12 @@ __pycache__/
 html/
 dist/
 .python-version
+build
 
 # Ignore some file types that may litter the root directory
-./*.txt
-./*.crt
-./*.sig
-./*.pem
-./*.sh
-./*.pub
+*.txt
+*.crt
+*.sig
+*.pem
+*.sh
+*.pub
```

Comment:
I think this will also ignore `sigstore/_store/*.{pem,crt}`, which isn't a problem for the already checked-in roots but might be a problem for future ones.

Reply:
4ae4097